media/css/firefox/whatsnew/whatsnew.scss

```diff
@@ -61,7 +61,6 @@ $image-path: '/media/protocol/img';
 // Main content section
 
 .main-content {
-    background: $color-off-black url('/media/img/firefox/whatsnew/whatsnew_background.svg') no-repeat;
     @include background-size(100%);
```

Comment:
We should also remove the `@include background-size(100%);` from this selector.

Reply:
Oh I missed it!